initCobraToolbox.m

```diff
@@ -614,7 +614,7 @@ function checkGit()
     end
     
     origLD = getenv('LD_LIBRARY_PATH');
-    newLD = regexprep(getenv('LD_LIBRARY_PATH'), [matlabroot '/bin/glnxa64:'], '');
+    newLD = regexprep(getenv('LD_LIBRARY_PATH'), [matlabroot '/bin/' computer('arch') ':'], '');
```

Comment:
Don't we have to use `filesep` here? Wouldn't this change of the `LD_LIBRARY_PATH` anyway only be set on `unix`?